### PR TITLE
chore(deps): remove ky

### PR DESCRIPTION
**Pull request by eglitise**
This PR replaces `ky` with `axios`. There's no need for two HTTP clients, and `ky` was outdated anyway.
Local session discovery was tested to still work.

### CODE REVIEW

File: app/renderer/actions/Session.js [hunk 1]
```diff
@@ -866,17 +866,16 @@ export function getRunningSessions () {
...
-          headers: {
-            'Authorization': `Basic ${btoa(`${username}:${accessKey}`)}`,
-            'content-type': HEADERS_CONTENT
-          }
-        }).json()
-        : await ky(`http${ssl ? 's' : ''}://${hostname}:${port}${adjPath}sessions`, {
-          headers: {'content-type': HEADERS_CONTENT}
-        }).json();
-      dispatch({type: GET_SESSIONS_DONE, sessions: res.value});
+        ? await axios({url, headers: {
```
Comment (mykola-mokhnach) [inline app/renderer/actions/Session.js:871]: we may only add the necessary header based on the condition:

```
await axios({url, headers: {
  'content-type': HEADERS_CONTENT,
  ...(username && accessKey ? {'Authorization': `Basic ${btoa(`${username}:${accessKey}`)}`} : {})
}});
```

**Review by mykola-mokhnach — APPROVED** [review]

**Review by jlipps — COMMENTED** [review]
I forget why I switched to Ky in the first place. I'm sure there was a reason. But this is no doubt OK.

**Comment by KazuCocoa** [comment]
That was probably to make https://inspector.appiumpro.com/ work. Axios did not work on browser, but worked on nodejs before. It could be wrong right now

**Comment by eglitise** [comment]
Yes, I just checked this in the browser as well (created a session via app, and tried to discover it via browser) - it worked fine.

**Comment by jlipps** [comment]
Ah yes that was why. Well as long as it works in the browser build, we should be fine!

**Pull request merged**